static/app/components/searchSyntax/parser.tsx

```diff
@@ -558,12 +561,14 @@ export class TokenConverter {
     items: [{separator: '', value: item1}, ...items.map(listJoiner)],
   });
 
-  tokenValueText = (value: string, quoted: boolean) => ({
-    ...this.defaultTokenFields,
-    type: Token.ValueText as const,
-    value,
-    quoted,
-  });
+  tokenValueText = (value: string, quoted: boolean) => {
+    return {
+      ...this.defaultTokenFields,
+      type: Token.ValueText as const,
+      value,
+      quoted,
+    };
+  };
```

Comment:
Let's leave the style for these as is
```suggestion
  tokenValueText = (value: string, quoted: boolean) => ({
    ...this.defaultTokenFields,
    type: Token.ValueText as const,
    value,
    quoted,
  });
```

Reply:
oh yes... the change is because I was debugging :)